packages/web3-core/types/index.d.ts

```diff
@@ -44,10 +46,13 @@ export class AbstractWeb3Module {
     transactionPollingTimeout: number;
     defaultGasPrice: string;
     defaultGas: number;
+    static readonly providers: Providers;
     readonly providers: Providers;
```

Comment:
the providers property is only a static property. I've changed this some minutes ago. I can do these changes also by myself. https://github.com/ethereum/web3.js/tree/fix/abstract-web3-module-providers

Reply:
Ok if you fix can remove the readonly typings in this branch as not on a computer!